### PR TITLE
changelog and updates for 7.1.0

### PR DESCRIPTION
**Pull request by baronfel**

### CODE REVIEW

**Comment by baronfel** [comment]
Putting this up so that we can talk about some things I'm seeing with the FSAC update:

* semantic tokenization seems like it's not triggering in some files?
* codeLens seem to not be returned?

**Comment by MangelMaxime** [comment]
With the release of FSAC v0.56.1, it is also be possible to include this PR in this release of Ionide: https://github.com/ionide/ionide-vscode-fsharp/pull/1760

Just wanted to point it out in case it was missed.

**Comment by Krzysztof-Cieslak** [comment]
v0.56.1 is completely broken so we still need to wait a bit here ;-)

**Comment by baronfel** [comment]
@MangelMaxime  I re-targeted your PR to this release branch and merged it, so it'll be included in the release

**Comment by baronfel** [comment]
We're fixing FSAC [here](https://github.com/fsharp/FsAutoComplete/pull/995) though before that happens.

**Comment by baronfel** [comment]
This version of FSAC looks and feels a lot better.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+### 7.1.0 - 19.08.2022
+* Update FSAC to 0.56.1 for [these changes](https://github.com/fsharp/FsAutoComplete/releases/tag/v0.56.1)
```
Comment (Krzysztof-Cieslak) [inline RELEASE_NOTES.md:2]: 0.56.2

Comment (baronfel) [reply RELEASE_NOTES.md]: good catch, updated.

**Comment by baronfel** [comment]
@MangelMaxime quick note - removing a file doesn't seem to clear out any diagnostics for it. might be interesting to run down, I'd expect it to be somewhere near the `textDocument/didClose` handling.

**Comment by baronfel** [comment]
This is good now, will merge and kick off a release later tonight.

**Pull request merged**